### PR TITLE
[CWS] skip useless work in the ptracer if no handler is defined for this syscall

### PR DESCRIPTION
**Pull request by paulcacheux**
### What does this PR do?

When using seccomp this is not an issue, but when we aren't we are calling the callback for a lot of useless syscalls with undefined handlers (futex, read, write etc) causing some unneeded slowdown.

### CODE REVIEW

**Review by safchain — APPROVED** [review]

File: pkg/security/ptracer/cws.go [hunk 1]
```diff
@@ -533,8 +538,7 @@ func ptrace(tracer *Tracer, probeAddr string, syscallHandlers map[int]syscallHan
 			}
 		case CallbackPostType:
 			syscallMsg, msgExists := process.Nr[nr]
-			handler, handlerFound := syscallHandlers[nr]
-			if handlerFound && msgExists && (handler.ShouldSend != nil || handler.RetFunc != nil) {
+			if msgExists {
```
Comment (spikat) [inline pkg/security/ptracer/cws.go:541]: I think we must keep the `&& (handler.ShouldSend != nil || handler.RetFunc != nil)` part

Comment (paulcacheux) [reply pkg/security/ptracer/cws.go:541]: the test is already done in the following ifs, I'm not sure I see the value is keeping this additional check

Comment (spikat) [reply pkg/security/ptracer/cws.go:541]: Yep you're right, I just saw it .. -_- sry ! LGTM

**Review by spikat — APPROVED** [review]

**Pull request merged**